backend/scripts/application_stop.sh

```
@@ -1,58 +1,51 @@
+# Graceful Shutdown은 BlockTraffic 단계에서 이미 시도됨
+# 이 단계에서는 남아있는 프로세스를 강제로 정리
 # ==========================================
+
 echo ""
-echo "☕ 1. Spring Boot 애플리케이션 종료 중..."
+echo "☕ 1. Spring Boot 애플리케이션 강제 종료 중..."
 
 if [ -f "app/coffee-shout.pid" ]; then
     PID=$(cat app/coffee-shout.pid)
 
     if ps -p $PID > /dev/null 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)